blog/good-compliance-programs-do-not-require-fancy-platforms.md

```diff
@@ -0,0 +1,235 @@
+I was also the auditor's point of contact with our company, funneling and prioritizing communications, and translating requirements into action items for our team.
+And I made sure to keep other key players up-to-date, or to push on them when deadlines closed in.
+
+I have worked with both [Strike Graph](https://www.strikegraph.com/), one of the compliance platforms, and [Martin Cozzi](https://www.marana.io/), an independent personalized consultant.
+
+From these experiences, here are my 5 key things that shaped our successful compliance program.
+
+### 1. Retain the help of an expert to get ramped up
+
+Unfortunately there is not a lot of room for trial and error when it comes to compliance programs, and a lot of pitfalls to overcome through self study.
+So, I definitely recommend getting the help of someone with experience.
+
+For startups and other small companies that do not have in-house experience yet, platforms like Strike Graph (for SOC2 at least) are actually not a bad way to go.
+They can help you get up and going for a lot less money than a personalized consultant.
+Strike Graph for example maintains a team of customer success agents who can steer you in the right direction by providing a lot of context and boilerplate policies for your organization to utilize.
+
+But just remember they are not in the business of understanding your business, and therefore they most likely will not be able to help with precision custom Control Statements that scale for you.
+And they may not be able to help accurately translate the expectations of your auditing partner into action steps.
+
+Also, something that all compliance platforms suffer from is that using an additional third party system means that it is yet another external system to login to and learn how to use.
+Think about it.
+Managing Control Statements and evidence outside of normal business tools, often means "out of sight, out of mind" and greatly reduces visibility into compliances processes internally.
+
+If you can afford a personalized consultant, such as Martin Cozzi, I would highly recommend that route.
+
+Consultants like Martin take the time to understand your business and the expectations of the auditing firm to craft a compliance strategy that is best suited to your company.
+This also means that they can translate auditor expectations into action steps for you.
+They can help write custom Control Statements that accurately reflect what you are doing and map them to compliance criteria.
+And my favorite part, consultants like Martin can show you how easy it is to manage compliance materials using the project management tools you **already use**.
+And if you don't use project management tools, then all you need is a Google spreadsheet.
+This is important because it shows once you are oriented in the compliance world, you don't need a fancy UI and tools to keep track of the state of your compliance efforts.
```

Comment:
"This is important because it shows once you are oriented in the compliance world"

I don't think that English makes sense.